auth.js

```diff
@@ -0,0 +1,105 @@
+const superagent = require("superagent");
+const superagentPrefix = require("superagent-prefix");
+const jwt = require("jsonwebtoken");
+const util = require("util");
+
+/**
+ * Auth class is used to make calls to getlabs api
+ */
+class Auth {
+
+  constructor(getlabsConifg) {
+    this.getlabsConifg = getlabsConifg;
+    this.oauthTokenResp = {
+      access_token: '',
+      refresh_token: ''
+    }
+  }
+
+  /**
+   * returns populated auth tokens
+   * @returns {*|{access_token: string, refresh_token: string}}
+   */
+  getAuthTokens() {
+    return this.oauthTokenResp;
+  }
+
+  /**
+   * returns the http client used to make external requests
+   * @param bearerToken: the Authorization bearer token value for said request
+   */
+  getHttpClient(bearerToken){
```

Comment:
I noticed the lack of space between the `)` and `{`. Can you make sure all the files are consistently formatted? It might be worth adding linting to the project (I probably should have done that when I started it).